Common.cs

```diff
@@ -63,6 +65,7 @@ public static GMapMarker getMAVMarker(MAVState MAV, GMapOverlay overlay = null)
                         itemr.Cog = MAV.cs.groundcourse;
                         itemr.Target = MAV.cs.target_bearing;
                         itemr.Nav_bearing = MAV.cs.nav_bearing;
+                        itemr.IsActive = MainV2.comPort.MAV == MAV;

```

Comment:
Is this required ?

Reply:
Not required for us, but part of my upstream PR.